mage_ai/data_cleaner/cleaning_rules/shared.py

```diff
@@ -0,0 +1,11 @@
+def build_action_variables(columns, column_types):
```

Comment:
You can just put it in BaseRule if you only use it in cleaning rules.

Reply:
Forgot about this, I moved this function into BaseRule